x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/job/retention/ExpiredModelSnapshotsRemoverTests.java

```diff
@@ -86,25 +91,22 @@ public void testRemove_GivenJobsWithoutRetentionPolicy() throws IOException {
 
         listener.waitToCompletion();
         assertThat(listener.success, is(true));
-        verify(client).search(any());
-        Mockito.verifyNoMoreInteractions(client);
```

Comment:
Why is this removed? Are there any new `client` interactions expected after your change?

Reply:
Yes, the filter client makes many more calls to the inner client.